src/components/OsnapV2/Faq.tsx

```diff
@@ -0,0 +1,70 @@
+import * as Accordion from "@radix-ui/react-accordion";
+import Link from "next/link";
+import { PlusMinus } from "./PlusMinus";
+
+export function Faq() {
+  const faqs = [
+    {
+      question: "How does UMA secure the governance process?",
+      answer:
+        "oSnap proposes successful governance votes to UMA, along with a bond. Anyone can dispute statements submitted to UMA by posting their own bond. This will trigger a vote, and if the dispute is successful, they will claim half the proposer's bond.",
```

Comment:
"they will claim" might be too ambiguous, maybe should be "the winner will claim" or "the disputer will claim"